.claude/scripts/comment-disclosure-drift.test.sh

```
@@ -82,6 +82,27 @@ else
   failures=$((failures + 1))
 fi
 
+echo "== consumer contract: --since re-verification is lane-scoped =="
+sweep_rule="$({
+  awk '
+    /^  🔴 \*\*On `--since`, read the findings/ { in_section = 1 }
+    in_section && /^  It reports \*\*positive evidence of agent authorship only\*\*/ { exit }
+    in_section { print }
+  ' "$constitution"
+} | tr '\n' ' ' | tr -s '[:space:]' ' ')"
+reverify_scope='Re-verify ONLY a body that is exactly `@cursor review`'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)